requirements.txt

```diff
@@ -14,6 +14,7 @@ django-apptemplates==1.3
 django-contrib-comments==1.8.0
 django-js-reverse==0.7.3
 django-queryset-csv==1.0.0               # https://github.com/azavea/django-queryset-csv/commits/master
+django-recaptcha==1.4.0
```

Comment:
You may already know about this, but while I was Googling to look at the environment variables available for this plugin, I stumbled upon https://github.com/kbytesys/django-recaptcha3.

Based on what I read, still seems like v2 (appears to be what we're using) is a safe bet.

Reply:
Yes. This looked like the most well used library with the smallest footprint and Google has no plans to shut off v2. I added the following quotes from the [FAQ](https://developers.google.com/recaptcha/docs/faq) to the NOTES section of the PR description

>reCAPTCHA v2 is not going away! We will continue to fully support and improve security and usability for v2.

>reCAPTCHA v3 is intended for power users, site owners that want more data about their traffic, and for use cases in which it is not appropriate to show a challenge to the user.